packages/api/package.json

```diff
@@ -8,7 +8,7 @@
   },
   "scripts": {
     "build-lambda-zips": "for x in $(ls dist); do (cd dist/${x} && rm -f lambda.zip && find . | xargs node ../../../../bin/zip.js lambda.zip && echo zipped ${x}); done",
-    "build": "../../node_modules/.bin/webpack",
+    "build": "node --max-old-space-size=4096 ../../node_modules/.bin/webpack",
```

Comment:
What does this do?

Reply:
This increases the memory available to webpack from the default to 4GB.    This was needed as (I believe) increased dependency size was causing the process to consume more memory than the default, resulting in failures on CI (but not local).